java/src/main/java/com/logicalclocks/hsfs/FeatureStore.java

```diff
@@ -68,11 +68,23 @@ public FeatureStore() {
   /**
    * Get a feature group object from the feature store.
    *
+   * <p>Getting a feature group from the Feature Store means getting its metadata handle so you can subsequently read
+   * the data into a Spark DataFrame or use the `Query`-API to perform joins between feature groups and create feature
```

Comment:
I have 2 problems with this sentence: 
- The metadata handle doesn't just let you read the data into a Spark DataFrame, it lets you interact with the feature group, i.e. you can write it as well.
- Even if there are code paths that let you get back a Spark DataFrame, I would not mention it on the FeatureStore object documentation, which is most likely going to stay the same when we'll have multiple implementations (Flink, ...)